types/index.d.ts

```diff
@@ -5314,8 +5135,8 @@ declare namespace math {
      * @param y Second value to xor
      */
     xor(
-      this: MathJsChain<number | BigNumber | Complex | Unit | MathCollection>,
-      y: number | BigNumber | Complex | Unit | MathCollection
```

Comment:
Function `xor` does not support `Fraction`, so we cannot use `MathType` here unfortunately.